src/librustc_parse/parser/diagnostics.rs

```diff
@@ -282,6 +286,34 @@ impl<'a> Parser<'a> {
         Err(err)
     }
 
+    fn check_too_many_raw_str_terminators(&mut self, err: &mut DiagnosticBuilder<'_>) -> bool {
+        let prev_token_raw_str = match self.prev_token {
```

Comment:
```suggestion
        let prev_token_raw_str = match self.prev_token.kind {
```
This should make the pattern matching a small bit nicer to read here.

Reply:
smart.